service/storage_service.cc

```diff
@@ -1142,13 +1143,18 @@ future<> storage_service::raft_state_monitor_fiber(raft::server& raft, gate::hol
                     get_ring_delay(),
                     _lifecycle_notifier,
                     _feature_service);
+            if (_feature_service.view_building_coordinator) {
+                auto vbc_ptr = std::make_unique<vbc::view_building_coordinator>(*as, _db.local(), *_group0, _sys_ks.local(), _messaging.local(), _topology_state_machine);
+                _view_building_coordinator = vbc::run_view_building_coordinator(std::move(vbc_ptr), _db.local(), *_group0);
```

Comment:
Why do you need to pass `_db.local()` and `_group0` to both `view_building_coordinator` and `run_view_building_coordinator`?

Reply:
In fact can you create `vbc_ptr` internally in `run_view_building_coordinator` like `run_topology_coordinator` does?